### PR TITLE
use environment variables for AWS creds instead of a local configuration file

### PR DESCRIPTION
**Pull request by sklegg**
another slight modification that should improve usability of this script inside a docker container

### CODE REVIEW

File: text2speech.py [hunk 1]
```diff
@@ -15,14 +16,8 @@
 
 separator = "="
 aws_properties = {}
-
```
Comment (zoglmannk) [inline text2speech.py]: Would you mind changing this to use environment variables but fall back to the aws.properties file if they aren't defined?

File: text2speech.py [hunk 2]
```diff
@@ -1,9 +1,10 @@
-#!/Users/kaz/anaconda3/bin/python
+#!/usr/bin/python3
```
Comment (zoglmannk) [inline text2speech.py]: Master has been updated to use the following line, which is the best option thus far.

#!/usr/bin/env python

**Comment by zoglmannk** [comment]
I will be happy to take a look again once the merge conflicts are handled. Luckily they aren't too bad.

**Comment by sklegg** [comment]
My latest changes should support both the aws.properties file and environment variables.

**Comment by zoglmannk** [comment]
The code changes look good. I will double check to make sure that everything still works this evening after work.

**Comment by zoglmannk** [comment]
Temporarily merging this to another branch. There is one small change that needs to be done before merging into master.

**Comment by zoglmannk** [comment]
This has now been merged into master

**Comment by zoglmannk** [comment]
Okay. everything should be good to go. Note for consistency, I changed AWS_SECRET_KEY to AWS_SECRET_ACCESS_KEY.

**Pull request merged**